llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -510,14 +510,14 @@ class InstrProfSymtab {
   /// an empty string.
   StringRef getFuncName(uint64_t FuncNameAddress, size_t NameSize);
 
-  /// Return function's PGO name from the name's md5 hash value.
-  /// If not found, return an empty string.
-  inline StringRef getFuncName(uint64_t FuncMD5Hash);
+  /// Return name of functions or global variables from the name's md5 hash
+  /// value. If not found, return an empty string.
+  inline StringRef getValueName(uint64_t ValMD5Hash);
 
-  /// Just like getFuncName, except that it will return a non-empty StringRef
-  /// if the function is external to this symbol table. All such cases
-  /// will be represented using the same StringRef value.
-  inline StringRef getFuncNameOrExternalSymbol(uint64_t FuncMD5Hash);
+  /// Just like getValueName, except that it will return a non-empty StringRef
+  /// if the value (function or global variable) is external to this symbol
+  /// table. All such cases will be represented using the same StringRef value.
+  inline StringRef getValueNameOrExternalSymbol(uint64_t ValMD5Hash);
```

Comment:
getFuncOrVarNameIfDefined().  Update comment to say 'External Symbol' is returned if not found.

Reply:
done.